### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by SashaViun**
[DEMO LINK](https://SashaViun.github.io/react_article/)

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -1 +1,42 @@
 // export const Article = () => ();
```
Comment (NVyrsta) [inline src/components/Article.jsx:1]: don't forget to delete redundant comments

**Review by NVyrsta — APPROVED** [review]
Well done 😎

File: src/components/Article.jsx (hunk 1 above)
Comment (alexandra-protyanova) [reply src/components/Article.jsx:1]: agree

**Review by alexandra-protyanova — APPROVED** [review]
GJ ⭐️

**Pull request closed without merging**